lib/components/InputFieldLabel/index.js

```diff
@@ -7,7 +7,6 @@ import theme from '../theme';
 
 const InputFieldLabel = styled.label`
   display: inline-block;
-  width: 100%;
```

Comment:
question: this seems like it might have broad consequences, is their a corresponding change elsewhere that keeps the visuals the same?

Reply:
Currently, the only two places we're utilizing `InputFieldLabel` is in the `TextField` component and the (new in this PR) `Toggle` component. (1) `TextField` doesn't require its label's box to expand wider than its label's content. Specifying a width here seems to be changing a default behavior for this display type for no reason. (2) Not specifying the width allows us to reuse the component in the new `Toggle` component.